app/src/main/kotlin/fr/o80/mostwanted/data/explanation/Explanation07.kt

```diff
@@ -13,14 +13,62 @@ fun Explanation07() {
         Modifier.fillMaxSize()
     ) {
         ExplanationText(
-            """Là l'utilisateur peut jouer avec l'écran.
+            """Nous allons maintenant rendre notre écran <b>interactif</b> et 
+                obtenir le comportement présent sur la maquette  ! Pour cela,
+                 reprenez le code précédemment écrit.
                 """
         )
         ExplanationText(
-            """Todo
-                     <b>Composables de mise en page</b>.
+            """Chaque modifier présente un attribut <b>clickable</b> qui 
+                permet de transmettre un comportement au clic sur le 
+                Composable associé.
+                """
+        )
+        ExplanationText(
+            """Nous voulons ici faire varier la taille des 4 coins de notre 
+                image. Pour ce faire, nous allons créer une variable dans le 
+                contenu de notre Composable. C'est celle-ci qui contiendra la taille 
+                souhaitée.
+                """
+        )
+        ExplanationText(
+            """Nous souhaitons que la valeur de notre variable soit 
+                persistante malgré la potentielle recomposition de notre 
+                écran. Nous allons donc la créer à l'aide d'un <b>remember</b>. 
+                Cette variable sera aussi modifiée afin de faire varier la 
+                forme de l'image. Il faut donc l'assigner avec un 
+                <b>mutableStateOf</b>.
+                """
+        )
+        Code("var corner by remember { mutableIntStateOf(0) }")
+        ExplanationText(
+            """La variable <b>corner</b> est ici initialisée à 0.
+                """
+        )
+        ExplanationText(
+            """Nous allons maintenant modifier le modifier <b>clip</b> que 
+                nous avons écrit dans l'exercice précédent pour lui attribuer
+                 la valeur de notre variable.
+                """
+        )
+        Code(".clip(RoundedCornerShape(corner))")
```

Comment:
Plutôt que de donnes la réponse, est-ce qu'on pourrait esasyer de donner des indices ?

```suggestion
        ExplanationText(
            """Voyez-vous comment faire varier la forme du clip en vous
                basant sur une autre forme que CircleShape ?
                """
        )
        ExplanationText(
            """Vous n'avez pas trouvé ? Que pensez-vous de RoundedCornerShape ?"""
        )
```